### DISCUSSION TITLE
Unit after transform

### DISCUSSION THREAD

**Discussion by jwjeremy** [pyproj4/pyproj#1096] [Q&A] [answered]
Hi, may i ask what is the unit after the transformation? 
For example, the transformer used is the following. 

> transformer = Transformer.from_crs('epsg:4326', \
>                                     "+proj=tmerc +datum=WGS84 +k=0.9996 +lon_0=143 +lat_0=39", \
>                                     always_xy=True)

Thank you!

**Comment by snowman2** [comment, marked as answer]
I recommend starting here: https://pyproj4.github.io/pyproj/stable/examples.html

The `CRS` class should help you find the destination units.

```python
from pyproj import CRS

CRS("+proj=tmerc +datum=WGS84 +k=0.9996 +lon_0=143 +lat_0=39")
```

**Reply by jwjeremy** [reply]
Thank you so much!
